.github/workflows/release.yml

```diff
@@ -65,3 +65,8 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GIT_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+  release-cli-assets:
+    needs: release
+    uses: ./.github/workflows/release-cli-assets.yml
+    secrets: inherit
```

Comment:
This should be a step after `Publish to npm` and runs conditionally only if publish was successful.